apps/src/gamelab/GameLabVisualizationColumn.jsx

```diff
@@ -151,6 +160,13 @@ class GameLabVisualizationColumn extends React.Component {
     );
   }
 
+  setDanceLabSong() {
+    if (document.getElementById("song_selector")) {
+      let song = document.getElementById("song_selector").value;
+      this.props.setSong(song);
```

Comment:
rather than peeking at the song_selector here, please make this take a parameter like `setDanceLabSong(songName)`, and have an `onChange` handler on the `select` element call `setDanceLabSong`.